Basic/DyanmicPrefabs/Assets/Scripts/01_Sparse loading with no latejoin/SparseLoadingNoLatejoinSample.cs

```diff
@@ -0,0 +1,132 @@
+using System;
```

Comment:
I'm not seeing discussions we had on when NOT to use dynamic prefabs (and just load assets as children of the parent NetworkObject). Could be worth at least talking about it or even have a scene for it.

Reply:
Stand-ins are out of scope - they aren't related to dynamic prefabs at all (they are just visualization-layer logic at most).

Parenting to a dynamically spawned prefab is the same as with the rest of the networkobjects - because NGO doesn't allow us to spawn-and-pray-that-clients-will-succeed - it wants us to preload stuff and be certain that the prefabs are in place on all peers before we spawn, so there shouldn't be a scenario when something is still loading for some peers - when spawn occurs, NGO will behave as though dynamic prefab is not any different from the one set up in Inspector.

With client-driven gameplay we still need the server to spawn the objects first, and this is only kosher if the prefabs have already been loaded - hence no difference with the rest of the client-driven gameplay.

We simply aren't allowed to do the minefield-style loading where some clients can fail when spawn has already occurred. This locks out all the potential craziness, making dynamic prefabs identical to the regular ones - when they are already safely spawned that is.

Sync and async addressables are again, irrelevant because of the same reason - by the time we spawn, all of the addressable loading has already concluded, so there would be no difference in implementation from the netcode perspective.